<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="VICTUS" failures="0" tests="1" name="runner.RegistrationRunnerClass" time="0.068" errors="1" timestamp="2025-03-20T03:09:13 IST" skipped="0">
  <testcase classname="runner.RegistrationRunnerClass" name="runScenario" time="0.068">
    <error message="Index 4 out of bounds for length 4" type="java.lang.IndexOutOfBoundsException">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index 4 out of bounds for length 4
at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:100)
at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:106)
at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:302)
at java.base/java.util.Objects.checkIndex(Objects.java:385)
at java.base/java.util.ArrayList.get(ArrayList.java:427)
at steps.RegistrationSteps.user_enter_the_following_details_as(RegistrationSteps.java:36)
at âœ½.User enter the following details as(file:///C:/Nikita/Testing%20OLD/Testing%20OLD%20Programming_2025/BDD_Cucumber_2025/src/test/resources/applicationfeature/Registration.feature:5)
]]>
    </error>
  </testcase> <!-- runScenario -->
</testsuite> <!-- runner.RegistrationRunnerClass -->
